app/services/find_out_of_sync_deploys_service.rb

```diff
@@ -22,6 +22,7 @@ def out_of_sync_apps_info
     out_of_sync_apps.map do |app|
       {
         name: app.name,
+        shortname: app.shortname,
```

Comment:
So from reading the commit message, you're saying the Release app link won't work for CKAN, Licence Finder etc...?

You may be able to programmatically derive it from the `puppet_name` in [repos.json](https://docs.publishing.service.gov.uk/repos.json)?

Reply:
I think I haven’t communicated my thinking in the commit message well.
I was trying to avoid touching the FindOutOfSyncDeploysService class to add a link to the Release app and thought about string manipulation on the values we were already passing (name or repo) to the PostOutOfSyncDeploysService. But neither `name.parametize` nor removing the `/alphagov/` prefix from `repo` will work for all the apps. The `shortname` is exactly what we need so no need to derive from `puppet_name`. Unless I misunderstood your comment.   

I have now squashed this into the previous commit and updated the commit message to mention why we need it.